lms/envs/test.py

```diff
@@ -525,6 +525,18 @@
 
 NOTES_DISABLED_TABS = []
 
+########################## EDRAAK FORUS ###############################
+
+# Enable Edraak ForUs app for tests
+FEATURES['ENABLE_EDRAAK_FORUS'] = True
```

Comment:
Let's keep the Edraak convention and name it like the following:

 - `FEATURES['EDRAAK_FORUS_APP']`.

Everywhere please 😃